### DISCUSSION TITLE
Convert `test.pkl`

### DISCUSSION THREAD

**Discussion by Xiangxu-0103** [open-mmlab/mmdetection3d#1898] [General]
Current repo do not support to update test dataset to `v2 format`. We may use an bool argument `test` [here](https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/tools/dataset_converters/update_infos_to_v2.py#L1036) to determine whether the dataset contain annotation information.

**Comment by ZCMax** [comment]
`test.pkl` conversion is not fully provided for now,  we will continuously update our conversion script.
